azure-pipelines/build.yml

```diff
@@ -43,6 +43,41 @@ steps:
     testRunTitle: Kusto.ServiceLayer.UnitTests
     arguments: '--configuration $(buildConfiguration) --collect "Code coverage"'
 
+- task: DotNetCoreCLI@2
+  displayName: Building Test Environment
+  inputs:
+    projects: '**/Microsoft.SqlTools.ServiceLayer.TestEnvConfig.csproj'
+
+- task: DownloadSecureFile@1
+  displayName: Download Test Environment Configuration
+  name: testEnvironmentPath
+  inputs:
+    secureFile: 'SQLConnectionInstances.xml'
+
+- task: DotNetCoreCLI@2
```

Comment:
We shouldn't be adding this stuff here - if anything the integration tests should just be referenced in the build-and-release.yml as a separate stage there.

That being said it's still reasonable that build.cmd should be building everything. Did you confirm whether it is/isn't building the integration tests project? Because fixing that up would be the first step and we can follow up with adding integration tests running during the product build and PR pipelines

Reply:
Please remove the rest of this for now, we shouldn't need it for just fixing the build.